torch/csrc/jit/codegen/cuda/ir_iostream.h

```diff
@@ -126,6 +127,10 @@ class TORCH_CUDA_API IRPrinter : public OptInConstDispatch {
   void printKernel(
       const std::vector<Expr*>& exprs,
       const std::string& kernel_name);
+
+ private:
+  std::unique_ptr<ThreadPredicateMap> thread_predicates_;
+  const ThreadPredicateMap& getThreadPredicateMap();
```

Comment:
please separate private methods from private data into separate private sections, ex. 
```
...
private:
  ... private methods ...

private:
  ... private data ...
```

Reply:
The rationale is to be able to locate the state easily, instead of having it mixed with methods.

Reply:
I'm not sure I agree here. It just has one field and one method. To make it a little easier to separate, I added an empty line between them.